releasewarrior/taskcluster.py

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env python
+
+import aiohttp
+import argparse
+import asyncio
+import async_timeout
+import posixpath
+from taskcluster.async import Index
+from taskcluster.async import Queue
+
+from releasewarrior.wiki_data import get_current_build_index
+
+
+HG_URL = 'https://hg.mozilla.org/'
+
+REPOS = ('releases/mozilla-beta', 'releases/mozilla-release',
+         'releases/mozilla-esr52', 'mozilla-central')
```

Comment:
since I took so long, I guess we need mozilla-esr60 now :)

Reply:
Aki just added support for esr60 in the templates, too, so I'll update for that.